modules/custom-yaml-configuration.adoc

```diff
@@ -0,0 +1,95 @@
+endpoints: <2>
+  # Serve plaintext HTTP only on port 8080
+  - listen: ":8080" <3>
+    # Backend protocols, possible values are 'http' and 'grpc'. If unset or empty, assume both.
+    protocols: <4>
+      - http
+    tls: <5>
+      # Disable TLS. If this is not specified, assume TLS is enabled.
+      disable: true
+  # Serve HTTP and  gRPC for sensors only on port 8444
+  - listen: ":8444" <3>
+    tls: <5>
+      # Which TLS certificates to serve, possible values are 'service' (For  service certificates that {product-title} generates)
+      # and 'default' (user-configured default TLS certificate). If unset or empty, assume both.
+      serverCerts: <6>
+        - default
+        - service
+      # Client authentication settings.
+      clientAuth: <7>
+        # Enforce TLS client authentication. If unset, do not enforce, only request certificates
+        # opportunistically.
+        required: true <8>
+        # Which TLS client CAs to serve, possible values are 'service' (CA for service
+        # certificates that {product-title} generates) and 'user' (CAs for PKI auth providers). If unset or empty, assume both.
+        certAuthorities: # if not set, assume ["user", "service"] <9>
+          - service
+----
+<1> Use `true` to disable exposure on the default port number `8443`.
+The default value is `false`, changing it to `true` might break existing functionality.
+<2> A list of additional endpoints for exposing Central.
+<3> The address and port number on which to listen. You must specify this value if you are using `endpoints`.
+You can use the format `port`, `:port`, or `address:port` to specify values.
+For example,
+* `8080` or `:8080` - listen on port `8080` on all interfaces.
+* `0.0.0.0:8080` - listen on port `8080` on all IPv4 (not IPv6) interfaces.
+* `127.0.0.1:8080` - listen on port `8080` on the local loopback device only.
+<4> Protocols to use for the specified endpoint.
+Acceptable values are `http` and `grpc`.
+If you do not specify a value, Central listens to both HTTP and gRPC traffic on the specified port.
+If you want to expose an endpoint exclusively for the RHACS portal, use `http`.
+However, you will not be able to use the endpoint for service-to-service communication or for the `roxctl` CLI, because these clients require both gRPC and HTTP.
+Red Hat recommends that you do not specify a value of this key, to enable both HTTP and gRPC protocols for the endpoint.
+If you want to restrict an endpoint to {product-title} services only, use the *clientAuth* option.
+<5> Use it to specify the TLS settings for the endpoint.
+If you do not specify a value, {product-title} enables TLS with the default settings for all the following nested keys.
+<6> Use `true` to disable TLS on the specified endpoint.
+The default value is `false`.
+When you set it to `true`, you cannot specify  values for `serverCerts` and `clientAuth`.
+<7> Specify a list of sources from which to configure server TLS certificates.
```

Comment:
Yeah this one looks like it matches up as the description for the `serverCerts` field. Not sure what the previous line was referencing. But if you can double check that each of these are matched up properly.